packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -409,19 +425,16 @@ function OrgNav({
           </Tooltip>
         )}
 
-        {!onPremEnabled &&
-          !isMobile &&
-          // @ts-ignore Property 'Appcues' does not exist on type 'Window & typeof globalThis'.ts(2339)
-          window.Appcues && (
-            <OrgNavLink
-              // @ts-ignore 'Cannot find name Appcues'
-              onClick={() => Appcues.show('-M4PVbE05VI91MJihJGv')} // eslint-disable-line no-undef
-              className={cx(rightLinkMargin, productTourColor)}
-              data-testid="org-nav-see-product-tour"
-            >
-              See Product Tour
-            </OrgNavLink>
-          )}
+        {showProductTour && (
```

Comment:
While we're in here, are we past `showProductTour` needing to be visible in the nav @DesignerDave?

Reply:
I think you're right! https://jira.mongodb.org/browse/PD-621 <- we've been clear since July 15th.

Reply:
@lazytype if you want to remove the product tour link and surrounding feel free, otherwise happy to include in my PR.

Reply:
@brookescarlett I'll let you include it. since I don't have as much context there's more chance I might accidentally screw something up